lib/scarlet/fmt.rb

```diff
@@ -30,22 +38,29 @@ def self.chop_msg msg
       i = 0 # increment
       bp = -1 # breakpoint
       loop do
+        should_break = false
```

Comment:
Not sure how this works really, but if you're splitting `\n`, why not just do a `str.split("\n")` first, then run your process over each string (first checking if `str.length > max_length`).